docker/readme.md

```diff
@@ -2,26 +2,109 @@
+Basic usage:
+```
+PS run-ubuntu1604.ps1 -ConfigPath C:\path\to\settings.json
 
-    -Detached - runs the image in detached mode (Docker's `-d`)
+Reading configuration from C:\path\to\settings.json.
+       _____                       _____  ____
+      |  __ \                     |  __ \|  _ \
+      | |__) |__ ___   _____ _ __ | |  | | |_) |
+      |  _  // _` \ \ / / _ \ '_ \| |  | |  _ <
+      | | \ \ (_| |\ V /  __/ | | | |__| | |_) |
+      |_|  \_\__,_| \_/ \___|_| |_|_____/|____/
 
-    -ConfigPath - allows to mount custom settings file from the host filesystem, absolute path is required
 
-    -Debug - runs shell in the interactive mode on the container (bash on Ubuntu, powershell on Windows)
+      Safe by default, optimized for efficiency
 
-    -BindPort 8080 - sets the port to which to bind container's RavenDB Server (default: 8080)
+ Build 40013, Version 4.0, SemVer 4.0.0-alpha-40013, Commit 1091651
+ PID 6, 64 bits
+ Source Code (git repo): https://github.com/ravendb/ravendb
+ Built with love by Hibernating Rhinos and awesome contributors!
++---------------------------------------------------------------+
+Listening to: http://0.0.0.0:8080
+Server started, listening to requests...
+Running as Service
 
-    -BindTcpPort 38888 - sets the port to which to bind container's RavenDB Server TCP port (default: 38888)
+```
 
-    -DbVolumeName - sets the Docker volume name used to persist data (default: ravendb)
+Once run RavenDB server should be exposed on port 8080 by default on the container.
+
+#### A bit on Docker network
+
+Once you run the image, to find out the Docker container IP address use:
+
+```
+> docker ps
+
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                 NAMES
+01ca48f18cfe        935f3b389940        "/bin/sh -c /opt/r..."   About an hour ago   Up About an hour    8080/tcp, 38888/tcp   keen_poitras
+```
+
+Then use the `CONTAINER ID` to inspect container properties:
+```
+> docker inspect 01ca48f18cfe
+...        
+```
+
+Among other fields there's *IPAddress* you can use to access Studio in your favorite web browser. Note, if you're running an Ubuntu container on Windows, please remember that your container is behind Hyper-V/VirtualBox NAT e.g. if you see `172.17.0.2` as an output of `docker inspect`, considering your docker NAT subnet address is `10.0.75.0` (See `Docker Settings -> Network -> Subnet Address`) and mask  `255.255.255.0`, then RavenDB Server is going to be accessible at `10.0.75.2:8080`.
+
+#### On Docker volumes usage
+
+Each of images above makes use of 2 volumes:
+
+- settings volume - holding RavenDB configuration, meant to be mounted from one of host's files
+
+    Ubuntu container: `/opt/raven-settings.json`
+
+    Windows container: `C:\raven-config` directory
+
+- databases volume - used for persistance of RavenDB data
+
+    Ubuntu container: `/databases`
```

Comment:
You can mount a host directory to a volume like we do with `settings.json` file. If not it's internal docker volume, persisted by docker - not recreated every time you run the image.

We can make an option for that in our run scripts, I'll add a ticket.